src/components/core/pivotTable/tests/ResizedColumnsStore.spec.ts

```diff
@@ -130,69 +149,264 @@ describe("ResizedColumnsStore", () => {
-            const resizedColumnsStore: any = new ResizedColumnsStore();
-            resizedColumnsStore.allMeasureColumnWidth = 42;
+        it("should return all measure column width when manuallyResizedColumns not exists", () => {
+            const resizedColumnsStore = new ResizedColumnsStore({}, 42, {});
+
             const columnMock = getFakeColumn({
+                colId: "m_0",
                 type: MEASURE_COLUMN,
             });
-            const expectedResult = { width: 42, source: ColumnEventSourceType.UI_DRAGGED };
+            const expectedResult = { width: 42 };
             const result = resizedColumnsStore.getManuallyResizedColumn(columnMock);
             expect(result).toEqual(expectedResult);
         });
 
         it("should return all measure column width when manuallyResizedColumns exists", () => {
-            const resizedColumnsStore = new ResizedColumnsStore({
-                m_0: {
-                    width: {
-                        value: 400,
+            const resizedColumnsStore = new ResizedColumnsStore(
+                {
+                    m_0: {
+                        width: {
+                            value: 400,
+                        },
+                    },
+                    a_4DOTdf: {
+                        width: {
+                            value: 200,
+                        },
                     },
-                    source: ColumnEventSourceType.UI_DRAGGED,
                 },
-                a_4DOTdf: {
-                    width: {
-                        value: 200,
+                42,
+            );
+
+            const columnMock = getFakeColumn({
+                colId: "m_0",
+                type: MEASURE_COLUMN,
+            });
+            const expectedResult = { width: 400 };
+            const result = resizedColumnsStore.getManuallyResizedColumn(columnMock);
+            expect(result).toEqual(expectedResult);
+        });
+
+        it("should not return manually resized measure column width is auto", () => {
+            const resizedColumnsStore = new ResizedColumnsStore({
+                m_0: { width: { value: "auto" } },
+                a_4DOTdf: { width: { value: 200 } },
+            });
+
+            const columnMock = getFakeColumn({
+                colId: "m_0",
+            });
+
+            const result = resizedColumnsStore.getManuallyResizedColumn(columnMock);
+            expect(result).toEqual(undefined);
+        });
+
+        it("should return weak column width when exist in internal weakMeasuresColumnWidths and not in manuallyResizedColumns", () => {
+            const resizedColumnsStore = new ResizedColumnsStore(
+                {
+                    a_4DOTdf: { width: { value: 200 } },
+                },
+                42,
+                weakMeasuresColumnWidths,
+            );
+
+            const columnMock = getFakeColumn({
+                colId: "m_0",
+                type: "MEASURE_COLUMN",
```

Comment:
It's sometimes used as `type: MEASURE_COLUMN` and sometimes as `type: "MEASURE_COLUMN"`. Is it an intention not to use imported constant?

Reply:
fixed